docs/new_branch_checklist.md

```diff
@@ -0,0 +1,147 @@
+     - [X] Update all files mentioning `master` to `<branch>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-pods@master`
+     - [X] Update images/manageiq-base/Dockerfile
+       - `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+       - `manageiq-release-<branch#>` to `manageiq-release-<nextbranch#>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-rpm_build@<branch>`
+     - [X] Update config/options.yml ref to `<branch>`
+     - [X] Ensure config/options.yml rpm.version is `<branch#>.0.0`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-rpm_build@master`
+     - [X] Update Dockerfile
+       - `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+       - `manageiq-release-<branch#>` to `manageiq-release-<nextbranch#>`
+     - [X] Update config/options.yml
+       - rpm.version to `<nextbranch#>.0.0`
+       - rpm_repository.content `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+       - rpms.manageiq and rpms.manageiq-release `<branch#>` in regex with `<nextbranch#>
+     - [X] Rename packages/manageiq-release/manageiq-<branch#>-<branch>.repo to manageiq-<nextbranch#>-<nextbranch>.repo
+     - [X] Update packages/manageiq-release/manageiq-<nextbranch#>-<nextbranch>.repo
+       - `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+     - [X] Update packages/manageiq-release/manageiq-release.spec
+       - Version to `<nextbranch#>.0`
+       - `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+       - Update `%changelog`
+     - [X] Update rpm_spec/changelog
+       - Update `%changelog`
+     - [X] Commit and push the changes
+   - [X] In `amazon_ssa_support@<branch>`
+     - [X] Update Gemfile to change the ref for manageiq-gems-pending
+     - [X] Commit and push the changes
+   - [X] In `container-amazon-smartstate@<branch>`
+     - [X] Update container-assets/Gemfile to change the ref for amazon_ssa_support and manageiq-gems-pending
+     - [X] Commit and push the changes
+   - [X] In `manageiq-providers-amazon@<branch>`
+     - [X] Update config/settings.yml agent_coordinator.docker_image from `latest` to `latest-<branch>`
+     - [X] Commit and push the changes
+
+5. Final development changes for the new branch
+
+   - [X] Mark the new branch as protected
+     - `bin/update_branch_protection.rb -b <branch>`
+   - [ ] Update miq-bot
+     - [ ] Add the new branch to each watched repo
+     - [ ] Update the unassignable and unremovable labels for the new branch
+
+6. Prepare for the next branch
+
+   - [X] Create the next milestone
+     - [X] `bin/update_milestone.rb --title <NextBranch> --due-on "MMM dd, YYYY"`
+   - [X] Update https://manageiq.org/roadmap with new column `<NextBranch>`
+
+7. Unpause the mirror script and adjust mirroring for the new branch
+
+
+---
+
+### Build
+
+- [ ] Create a build machine for the new branch
+- [ ] Create a yum repository for the next branch for master nightlies
```

Comment:
This should happen on first build

Reply:
looks like downstream is already expecting this so we'll have to move it up the list earlier?

Reply:
Delete based on https://github.com/ManageIQ/manageiq-release/pull/188/files#r581176061
```suggestion
```